client/rest-high-level/src/test/java/org/elasticsearch/client/ESRestHighLevelClientTestCase.java

```diff
@@ -76,16 +76,42 @@ protected static RestHighLevelClient highLevelClient() {
         }
     }
 
+    /**
+     * Executes the provided request using either the sync method or its async
+     * variant, both provided as functions. This variant is used for when the
+     * request does not have a body (data is conveyed by path and headers).
```

Comment:
this javadoc is not quite correct. There is really used when there is no request object

Reply:
Agreed, thanks for pondering about it!